src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -667,8 +737,42 @@ private static SeriesTitleInfo GetSeriesTitleInfo(string title)
 
         private static ParsedTrackInfo ParseMatchMusicCollection(MatchCollection matchCollection)
         {
-            throw new NotImplementedException();
+            var artistName = matchCollection[0].Groups["artist"].Value.Replace('.', ' ').Replace('_', ' ');
```

Comment:
This might become problematic with Artists such as Will.I.Am (See our parsing code for how we did that)